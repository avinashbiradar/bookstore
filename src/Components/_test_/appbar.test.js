import React from 'react'; 
import {shallow,mount} from 'enzyme';
import Logo from "../AppBar/AppBar"; 
import logoImage from "../assests/education.svg";

import Badge from "@material-ui/core/Badge";

describe('<Logo /> with no props', () => {
    const container = shallow(<Logo />);
    it('should match the snapshot', () => {
      expect(container.html()).toMatchSnapshot();
    });
  });

describe("<Logo />", () => {
    it("renders an image", () => {
        const logo = shallow(<Logo />);
        expect(logo.find("img").prop("src")).toEqual(logoImage);

     });
 });

//  describe("props testing ", () => {
//     it("renders an image", () => {
//         const logo =shallow(<Logo  totalCartItem={2} />);
//         // expect(logo.find("#badge-cart"));
//         // const result = logo.find("#badge-cart").contains(5)
//         // console.log("result",result.debug())
//          const debug = Logo.find(Badge).debug()
//          console.log("debug",debug)
//      });
//  });

