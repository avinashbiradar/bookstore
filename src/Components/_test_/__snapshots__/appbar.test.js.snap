// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> with no props should match the snapshot 1`] = `"<header class=\\"MuiPaper-root MuiAppBar-root MuiAppBar-positionFixed MuiAppBar-colorPrimary makeStyles-AppBar-1 mui-fixed MuiPaper-elevation4\\"><div class=\\"MuiToolbar-root MuiToolbar-regular makeStyles-toolBar-2 MuiToolbar-gutters\\"><div class=\\"makeStyles-leftOptions-3\\"><div class=\\"makeStyles-title-6\\"><img class=\\"makeStyles-titleLogo-7\\" src=\\"education.svg\\"/><h6 class=\\"MuiTypography-root makeStyles-titleName-8 MuiTypography-h6\\">Bookstore</h6></div><div class=\\"makeStyles-search-5\\"><div class=\\"makeStyles-searchIcon-9\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z\\"></path></svg></div><div class=\\"MuiInputBase-root\\"><input type=\\"text\\" placeholder=\\"Searchâ€¦\\" class=\\"MuiInputBase-input makeStyles-inputInput-12\\"/></div></div></div><div class=\\"makeStyles-rightOptions-4\\"><svg class=\\"MuiSvgIcon-root makeStyles-buttonSearch-11\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z\\"></path></svg><svg class=\\"MuiSvgIcon-root makeStyles-profileButton-14\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\" aria-controls=\\"fade-menu\\" aria-haspopup=\\"true\\"><path d=\\"M12 5.9c1.16 0 2.1.94 2.1 2.1s-.94 2.1-2.1 2.1S9.9 9.16 9.9 8s.94-2.1 2.1-2.1m0 9c2.97 0 6.1 1.46 6.1 2.1v1.1H5.9V17c0-.64 3.13-2.1 6.1-2.1M12 4C9.79 4 8 5.79 8 8s1.79 4 4 4 4-1.79 4-4-1.79-4-4-4zm0 9c-2.67 0-8 1.34-8 4v3h16v-3c0-2.66-5.33-4-8-4z\\"></path></svg><button class=\\"MuiButtonBase-root MuiIconButton-root makeStyles-cartButton-13\\" tabindex=\\"0\\" type=\\"button\\"><span class=\\"MuiIconButton-label\\"><span class=\\"MuiBadge-root\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M15.55 13c.75 0 1.41-.41 1.75-1.03l3.58-6.49c.37-.66-.11-1.48-.87-1.48H5.21l-.94-2H1v2h2l3.6 7.59-1.35 2.44C4.52 15.37 5.48 17 7 17h12v-2H7l1.1-2h7.45zM6.16 6h12.15l-2.76 5H8.53L6.16 6zM7 18c-1.1 0-1.99.9-1.99 2S5.9 22 7 22s2-.9 2-2-.9-2-2-2zm10 0c-1.1 0-1.99.9-1.99 2s.89 2 1.99 2 2-.9 2-2-.9-2-2-2z\\"></path></svg><span class=\\"MuiBadge-badge WithStyles(ForwardRef(Badge))-badge-16 MuiBadge-anchorOriginTopRightRectangle MuiBadge-invisible\\"></span></span></span></button></div></div></header>"`;
